bot/src/keeper.ts

```diff
@@ -40,6 +55,14 @@ const checkAndParticipateIfPossible = async function (auction: AuctionInitialInf
         return;
     }
 
+    // check if collateral is white listed
+    if (!isCollateralInWhitelist(auction.collateralSymbol)) {
+        console.info(
+            `keeper: auction "${auction.id}" was not executed because "${auction.collateralSymbol}" is not in the whitelist.`
+        );
+        return;
+    }
```

Comment:
As specified in the issue, this should be applied not only for the keeper, but for the twitter bot as well. Hint: we shouldn't even fetch auctions of the non-whitelisted collaterals